spec/unit/pops/types/p_object_type_spec.rb

```diff
@@ -448,7 +444,7 @@ def parse_object(name, body_string)
         equality => [a,b]
       OBJECT
       expect { parse_object('MyObject', obj) }.to raise_error(Puppet::ParseError,
-        "equality is referencing constant attribute 'b'")
+        "MyObject equality is referencing constant attribute MyObject[b]")
```

Comment:
think it needs to say why that is wrong "reference to constant not allowed in equality" or somesuch.